Framework/Kernel/test/DataServiceTest.h

```diff
@@ -6,6 +6,9 @@
 #include <cxxtest/TestSuite.h>
 #include <Poco/NObserver.h>
 #include <boost/make_shared.hpp>
+
+#include <iostream>
```

Comment:
Can we remove this header? It's a big header and can easily get left around when it's not required.